### ISSUE TITLE
Offsetting by thirds

### ISSUE THREAD

**Issue by gukj** [dhg/Skeleton#161]
I think the css would be more complete with the addition of offset-by-one-third, and offset-by-two-thirds

For desktop:

```
.container .offset-by-one-third             { padding-left: 320px; }
.container .offset-by-two-thirds            { padding-left: 640px; }
```

768px < screen < 959px

```
.container .offset-by-one-third             { padding-left: 256px; }
.container .offset-by-two-thirds            { padding-left: 512px; }
```

For mobile

```
.container .offset-by-one-third,
.container .offset-by-two-thirds { padding-left: 0; }
```

**Comment by joahg** [comment]
:+1:

**Comment by dhg** [comment]
This is fixed in v2.0. One/two thirds offsets now exist :)
